modules/fedimint-wallet-client/src/lib.rs

```diff
@@ -981,6 +983,11 @@ impl WalletClientModule {
             stream! {
                 yield DepositStateV2::WaitingForTransaction;
 
+                retry(
+                    "subscribe script history",
+                    background_backoff(),
+                    || stream_rpc.watch_script_history(&stream_script_pub_key)
+                ).await.expect("Will never give up");
```

Comment:
Same comment as https://github.com/fedimint/fedimint/pull/7680/files#r2316574083. Not sure why this needs to be in a loop.